stylesheets/components/buttons.css

```diff
@@ -0,0 +1,151 @@
+.btn-wrapper {
+  background-color: var(--white-lilac);
+  border-radius: 6px;
+  border: 1px solid var(--pastel-grey);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2rem;
+  padding: 2rem;
+}
+/* general styles for all buttons */
+.btn {
+  border-radius: 6px;
+  border: 2px solid transparent;
+  padding: 0.7rem 1.2rem;
+  font-weight: 700;
+}
+.btn:hover {
+  cursor: pointer;
```

Comment:
Everything is good to go just a small suggestion like you can add this property in the .btn class no need to add hover pseudo for cursor pointer. specifying it in .btn class will do the work

Reply:
I didn't know we could it this way also, I've done the changes. Thanks for letting me know!